### PR TITLE
bpo-39849: Enable assertions in _testcapimodule.c and _testinternalcapi.c

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue39849

### CODE REVIEW

File: Modules/_testcapimodule.c [hunk 1]
```diff
@@ -11,6 +11,8 @@
    The Visual Studio projects builds _testcapi with Py_BUILD_CORE_MODULE
    macro defined, but only the public C API must be tested here. */
 #undef Py_BUILD_CORE_MODULE
```
Comment (vstinner) [inline Modules/_testcapimodule.c:14]: nitpick: add an empty line.

Comment (shihai1991) [reply Modules/_testcapimodule.c:14]: done

**Review by vstinner — APPROVED** [review]
LGTM.

**Review by corona10 — APPROVED** [review]
LGTM, 
I 've also checked on my local machine, no warning is now showing.

**Comment by shihai1991** [comment]
Thanks,  folks ;)

**Pull request merged**